### PR TITLE
Add Finnish payment terms

### PR DESCRIPTION
**Pull request by jarmokortetjarvi**
Add common Finnish payment terms and helper fields for displaying them in correct order (sequence) and searching (code) without using error-prone fields 'name' or 'note'

### CODE REVIEW

File: l10n_fi_payment_terms/data/payment_terms_finnish.xml [hunk 1]
```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+    
+        <!--
+        	Payment terms
+        -->
+        
+        <!-- Delete the defaults -->
+        <delete model="account.payment.term" id="account.account_payment_term_immediate"/>
```
Comment (mlaitinen) [inline l10n_fi_payment_terms/data/payment_terms_finnish.xml]: A bit hazardous for non-fresh installations. I'd leave the original payment terms untouched.

File: l10n_fi_payment_terms/models/payment_term.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class PaymentTerm(models.Model):
+
+    # 1. Private attributes
+    _inherit = 'account.payment.term'
+
+    # 2. Fields declaration
+    _order = 'sequence, name'
+
+    sequence = fields.Integer('Sequence')
+    code = fields.Char('Unique code')
```
Comment (mlaitinen) [inline l10n_fi_payment_terms/models/payment_term.py:14]: I'm not sure if these fields are necessary. Nevertheless I'm not opposed to keeping them.

File: l10n_fi_payment_terms/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+* Jarmo Kortetjärvi <jarmo.kortetjarvi@tawasta.fi>
+
+Maintainer
+----------
+
+.. image:: http://tawasta.fi/templates/tawastrap/images/logo.png
+   :alt: Oy Tawasta OS Technologies Ltd.
+   :target: http://tawasta.fi/
+
```
Comment (mlaitinen) [inline l10n_fi_payment_terms/README.rst]: OCA should be a maintainer as well

**Review by mlaitinen — CHANGES_REQUESTED** [review]
Great job. I'd skip the removal of the original payment terms.

File: l10n_fi_payment_terms/models/payment_term.py (hunk 2 above)
Comment (jarmokortetjarvi) [reply l10n_fi_payment_terms/models/payment_term.py:14]: I think these aren't absolutely necessary, but they are useful as alphabetical ordering is confusing in this instance. They should, however, be introduces in another module than a localization one.

File: l10n_fi_payment_terms/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+import payment_term
```
Comment (dreispt) [inline l10n_fi_payment_terms/models/__init__.py]: `from . import payment_term`

File: l10n_fi_payment_terms/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+import models
```
Comment (dreispt) [inline l10n_fi_payment_terms/__init__.py]: Use relative import: `from . import models`

**Review by dreispt — CHANGES_REQUESTED** [review]

File: l10n_fi_payment_terms/models/payment_term.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+    # 4. Compute and search fields, in the same order that fields declaration
+
+    # 5. Constraints and onchanges
+
+    # 6. CRUD methods
+
+    # 7. Action methods
+
+    # 8. Business methods
```
Comment (dreispt) [inline l10n_fi_payment_terms/models/payment_term.py]: Remove unneeded comments.

File: l10n_fi_payment_terms/models/payment_term.py (hunk 2 above)
Comment (dreispt) [reply l10n_fi_payment_terms/models/payment_term.py:14]: On one hand it is not strictly necessary, on another hand IMHO these "code" fields can be useful.
So no concern on having it here.

File: l10n_fi_payment_terms/models/payment_term.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+# 1. Standard library imports:
```
Comment (dreispt) [inline l10n_fi_payment_terms/models/payment_term.py]: Please remove all these guideline comments

**Review by dreispt — APPROVED** [review]

**Review by mlaitinen — APPROVED** [review]

**Pull request merged**